service-coordtransform/src/main/java/fi/nls/paikkatietoikkuna/coordtransform/CoordinateTransformationActionHandler.java

```diff
@@ -50,30 +107,87 @@ public void init() {
         if (endPoint == null) {
             endPoint = PropertyUtil.getNecessary(PROP_END_POINT);
         }
+        lineSeparators.put("win","\r\n");
+        lineSeparators.put("mac","\n");
+        lineSeparators.put("unix","\r");
+
+        coordinateSeparators.put("space", " ");
+        coordinateSeparators.put("tab", "\t");
+        coordinateSeparators.put("comma", ",");
     }
 
     @Override
     public void handleAction(ActionParameters params) throws ActionException {
+        hasMoreCoordinates = false;
```

Comment:
This won't work. Can't store state in since another request to this same Handler will set the value to false once again.